doc/lua_api.txt

```diff
@@ -414,6 +414,101 @@ the word "`alpha`", then each texture pixel will contain the RGB of
 `<color>` and the alpha of `<color>` multiplied by the alpha of the
 texture pixel.
 
+Particle blend
+--------------
+The blend function is defined by an integer number and there are
+a huge number of acceptable blend modificators.
+The color channels (RGB) of resulting pixel are calculated using this formula:
+
+	red = source_red * source_factor + destination_red * destination_factor
+
+
+Some examples:
+
+Disabled blend (default):
+
+	blend_type = 0,
```

Comment:
Please can we have another table entry to use
`blend_type = minetest.blend_type.none,`
here?